src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -199,33 +199,35 @@ private static void checkGenomicsDBAgainstExpected(final String workspace, final
     }
 
     @Test
-    public void testSampleMappingFileInsteadOfVCFs() throws IOException {
-        final File sampleNameFile = createTempSampleMapFile();
+    public void testSampleMappingFileInsteadOfVCFsWithMultipleBatches() throws IOException {
+        final File sampleNameFile = createOutOfOrderTempSampleMapFile();
```

Comment:
New test case(s) should supplement, not replace, existing ones. I think what you should do here is create a `DataProvider` that generates test cases with all combinations of:

-batch sizes 0, 1, 2, 3, 4
-samples in order vs. samples out of order
-sample map file vs. -V